Loop/Managers/DeviceDataManager.swift

```diff
@@ -566,9 +627,10 @@ final class DeviceDataManager {
     func generateDiagnosticReport(_ completion: @escaping (_ report: String) -> Void) {
         self.loopManager.generateDiagnosticReport { (loopReport) in
 
-            self.alertManager.getStoredEntries(startDate: Date() - .hours(48)) { (alertReport) in
+            let logDurationHours = 84.0
```

Comment:
Heh, I thought that it might be too when I saw it a while after changing it, but it's to ensure a full pod cycle worth of messages is included in the issue report. I'd like to eventually remove device logs from the issue report, and have support people use the device logs that are exported in the Critical Event Logs, but there are still tools and things depending on getting this data from the issue report.